spec/custom_matchers.rb

```diff
@@ -0,0 +1,12 @@
+RSpec::Matchers.define :belong_to do |expected|
+  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+end
```

Comment:
Do you really believe you are the first rubyist that needs matcher "same arrays, ignoring order"? ;)
Standard `match_array` matcher will do.